ding/reward_model/max_entropy_irl_model.py

```diff
@@ -70,24 +83,25 @@ def train(self, expert_demo: torch.Tensor, samp: torch.Tensor, iter, step):
         prob = samp['prob'].unsqueeze(-1)
         #print(len(cost_samp))
```

Comment:
I suppose it will be nice to print some logs just as RL training part. Maybe it is nice to uncomment some important outputs to let people see their IRL part training. After this, please delete all irrelevant comments.